docs/content/docs/recipes/teams.md

```diff
@@ -63,21 +63,21 @@ This tutorial assumes that you are familiar with the [Fluid Framework Overview](
 // TODO 6: Define get container function.
 ```
 
-1. Replace `TODO 1:` with the following code. Note that we are exporting the constant because we will be using it to append to the `contentUrl` in the Microsoft Teams settings and parsing the container ID in the content page. It's a common pattern to store important query parameter keys as constants, rather than typing the raw string each time. This is also a good place to define the DDS keys.
+1. Replace `TODO 1:` with the following code. Note that the constant is being exported because it will be used to append to the `contentUrl` in the Microsoft Teams settings and later for parsing the container ID in the content page. It's a common pattern to store important query parameter keys as constants, rather than typing the raw string each time. This is also a good place to define the DDS keys.
```

Comment:
```suggestion
1. Replace `TODO 1:` with the following code. Note that the constant is being exported because it will be used to append to the `contentUrl` in the Microsoft Teams settings and later for parsing the container ID in the content page. It's a common pattern to store important query parameter keys as constants, rather than typing the raw string each time.
```

Unless your code sample illustrates this, don't mention it.